Tests/AFNetworkActivityLoggerTests.m

```diff
@@ -265,5 +265,52 @@ - (void)testThatIndividualLoggerIsNotCalledWhenLoggerIsNilledOut {
     [manager invalidateSessionCancelingTasks:YES];
 }
 
+- (void)testThatResponseSerializerIsAFHTTPResponseSerializerAndResponseBodyIsText {
+    NSURL *baseURL = [NSURL URLWithString:@"https://httpbin.org"];
+    AFHTTPSessionManager *manager = [[AFHTTPSessionManager alloc] initWithBaseURL:baseURL];
+    manager.responseSerializer = [AFHTTPResponseSerializer serializer];
+    AFNetworkActivityTestLogger *testLogger = [AFNetworkActivityTestLogger new];
+    
+    XCTestExpectation *expectation = [self expectationWithDescription:@"Finish Block Should Be Called"];
+    [testLogger setFinishBlock:^(NSURLSessionTask *task, id responseObject, NSTimeInterval elpasedTime, NSError *error) {
+        [expectation fulfill];
+    }];
+    [self.logger addLogger:testLogger];
+    [self.logger setLogLevel:AFLoggerLevelDebug];
+    [self.logger startLogging];
+    
+    [manager
+     GET:@"ip"
+     parameters:nil
+     progress:nil
+     success:nil
+     failure:nil];
+    [self waitForExpectationsWithTimeout:10.0 handler:nil];
+    [manager invalidateSessionCancelingTasks:YES];
+}
+
+- (void)testThatResponseSerializerIsAFHTTPResponseSerializerAndResponseBodyIsNotText {
```

Comment:
add the test case: when _manager.responseSerializer = [AFHTTPResponseSerializer serializer];_ and the http response body is not text, for example, the response body is a image